config/vufind/KohaRest.ini

```diff
@@ -151,3 +151,8 @@ extraFields = item-issue:acceptTerms:pickUpLocation
 ; config.ini defaults when Voyager is used for authentication.
 ;pattern = "numeric"
 ;hint = "Your optional custom hint can go here."
+
+; Show holds transaction history, default is set to false.
```

Comment:
Sorry, changing approval. The comment should be revised. The comment above the section should be about the full section which might contain other settings as well. Then right above the `enabled = true` you'd have something like `; Whether the transaction history functioanlity is enabled. Default is false.`. This does not involve holds.

Reply:
I have copied the section from demo.ini so I hope it is ok.